docker-compose.yaml

```diff
@@ -10,6 +10,14 @@ services:
       POSTGRES_DB: scheduler
       POSTGRES_USER: root
 
```

Comment:
I think redis can be added . too

Reply:
or, an another docker-compose-up config can be created that boots up redis and mysql instead of postgres and couchbase

Reply:
I didn't any changes for this. Because test user might need all of tools to test. But I added redis.